### PR TITLE
static_cast to avoid warning

### PR DESCRIPTION
**Pull request by afabri**
This should fix this [warning](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.9-Ic-22/Surface_mesh_deformation/TestReport_jtournois_x64_Cygwin-Windows8_MSVC2015-Release-64bits.gz)

### CODE REVIEW

File: Surface_mesh_deformation/test/Surface_mesh_deformation/Cactus_deformation_session_OpenMesh.cpp [hunk 1]
```diff
@@ -39,7 +39,7 @@ void compare_mesh(const Mesh& mesh_1, const Mesh& mesh_2)
   {
     total_dif = total_dif + (get(ppmap_1, *it_1) - get(ppmap_2, *it_2));
   }
-  double average_mesh_dif = squared_length(total_dif / num_vertices(mesh_1));
+  double average_mesh_dif = squared_length(total_dif / static_cast<float>(num_vertices(mesh_1)));
```
Comment (lrineau) [inline Surface_mesh_deformation/test/Surface_mesh_deformation/Cactus_deformation_session_OpenMesh.cpp]: This code is strange. `total_diff` is a point, with 3D coordinates. The following should be better:
```C++
const std::size n = num_vertices(mesh_1);
double average_mesh_dif = squared_length(total_dif) / n / n;
```

**Comment by lrineau** [comment]
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.9-Ic-23/Surface_mesh_deformation/TestReport_jtournois_x64_Cygwin-Windows8_MSVC2015-Debug-64bits.gz
> D:\CGAL\test\CGAL-4.9-Ic-23\include\CGAL/Surface_mesh_deformation.h(1405): warning C4244: 'argument': conversion from 'double' to 'const float', possible loss of data
D:\CGAL\test\CGAL-4.9-Ic-23\include\CGAL/Surface_mesh_deformation.h(1341): note: while compiling class template member function 'void CGAL::Surface_mesh_deformation<Mesh,CGAL::Default,CGAL::Default,SPOKES_AND_RIMS,CGAL::Default,CGAL::Default,CGAL::Default,CGAL::Default>::update_solution_spokes_and_rims(void)'
D:\CGAL\test\CGAL-4.9-Ic-23\include\CGAL/Surface_mesh_deformation.h(1268): note: see reference to function template instantiation 'void CGAL::Surface_mesh_deformation<Mesh,CGAL::Default,CGAL::Default,SPOKES_AND_RIMS,CGAL::Default,CGAL::Default,CGAL::Default,CGAL::Default>::update_solution_spokes_and_rims(void)' being compiled
D:\CGAL\test\CGAL-4.9-Ic-23\cmake\platforms\x64_Cygwin-Windows8_MSVC2015-Debug-64bits\test\Surface_mesh_deformation\Cactus_deformation_session_OpenMesh.cpp(104): note: see reference to class template instantiation 'CGAL::Surface_mesh_deformation<Mesh,CGAL::Default,CGAL::Default,SPOKES_AND_RIMS,CGAL::Default,CGAL::Default,CGAL::Default,CGAL::Default>' being compiled

**Comment by lrineau** [comment]
Ping @afabri

**Comment by afabri** [comment]
It is solved in PR #1021.
If you prefer I can apply the change in this ( == 1011) PR.

**Comment by lrineau** [comment]
I will test and merge both PR at the same time.

**Pull request merged**